README.md

```diff
@@ -1,96 +1,24 @@
-# GitHub Gems: Driving Open-Source Investments With Data
+# Christopher Akey
 
-Welcome to the GitHub Gems project! This project hosts a data analytics pipeline that enables smarter investment decisions by measuring the popularity of open-source repos on Github.
+What insights are you hoping to get?
```

Comment:
All the content looks great! I'll encourage switching to titles that are more like real-life product documentation e.g. "Overview", "Metrics", etc.